packages/expo-brightness/src/Brightness.ts

```diff
@@ -8,6 +8,19 @@ export enum BrightnessMode {
   MANUAL = 2,
 }
 
+export const PermissionsStatus = {
+  GRANTED: 'granted',
+  UNDETERMINED: 'undetermined',
+  DENIED: 'denied',
+} as const;
+
+export type PermissionsResponse = {
+  status: typeof PermissionsStatus[keyof typeof PermissionsStatus];
+  expires: 'never' | number;
+  granted: boolean;
+  canAskAgain: boolean;
+};
+
```

Comment:
It might be better to have these types defined in `unimodules-permissions-interface` and re-export them in modules using them.